### DISCUSSION TITLE
This site can’t provide a secure connection

### DISCUSSION THREAD

**Discussion by qublaidev** [vercel/community#5099] [Help] [answered]
### Summary

I am using a vercel generated domain like "xxxxx.vercel.app" and all of a sudden I get ERR_SSL_PROTOCOL_ERROR, I didn't changed a thing in like a month.

I think this might be related to [https://www.reddit.com/r/nextjs/comments/1418l25/vercelcom_blacklisted_by_a_major_isp/](vercelcom_blacklisted_by_a_major_isp)
because when I use a VPN I can access the site

### Steps to Reproduce

1.open https://gradient-guru.vercel.app/ in TR and receive error
2.open it with VPN, no error

**Comment by fossejc** [comment]
I have the same issue when using my wifi provider but no errors when using my 4g network.

Where you able to fix this issue ?

Thanks,
Jean-Charles

**Comment by deep-wb** [comment]
The ERR_SSL_PROTOCOL_ERROR is a frequently encountered message indicating a hiccup in the SSL/TLS (Secure Sockets Layer/Transport Layer Security) protocol during an attempt to establish a secure connection. This common issue arises in popular web browsers like Google Chrome, Mozilla Firefox, or Microsoft Edge when users aim to access a secure website.

I encourage you to give this blog a thorough read. It holds the key to a comprehensive understanding of this error, ensuring you find the precise solution you need for a seamless browsing experience [read here](https://bit.ly/SSL_Error).

**Comment by swarnava** [comment, marked as answer]
Hi there,

This sounds like a VPN / ISP specific block which is beyond our control. We will recommend reaching out to respective ISP provider or VPN suppoirt for more information on this.
